Automattic/woocommerce-payments
# remove unnecessary tracks events for acceptDispute submit success/fail

Fixes #8064

#### Changes proposed in this Pull Request

This PR removes two events that are fired as part of request handling, and don't represent user actions.

- `wcadmin_wcpay_dispute_accept_success`
- `wcadmin_wcpay_dispute_accept_failed`

If we want to measure merchant behaviour when wrangling disputes, there are other events we can use (which track true user actions): 

- Merchant views transaction details page => accepts dispute: `wcadmin_page_view[page=transaction_details] > wcpay_dispute_accept_click`

#### Testing instructions
- Ensure your site is set up to send tracks events and you have a way to view fired events
- View a transaction details page for a live dispute
- Accept the dispute
- Ensure the `wcadmin_wcpay_dispute_accept_*` events are not fired
- Explore various dispute flows and ensure that relevant user actions are captured with events (e.g. merchant action to accept a disputes)


-------------------

#### Checklist


**Post merge**
